specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -8527,6 +8548,10 @@
         "licenseType": {
           "type": "string",
           "description": "The license type, which is for bring your own license scenario."
+        },
+        "scheduledEventsProfile" : {
+          "$ref":"#/definitions/ScheduledEventsProfile",
+          "description": "Specifies Scheduled Event related configurations."
         }
```

Comment:
Is this a required property?

Reply:
Not required

Reply:
Thanks for confirming.

---
In reply to: [309897834](https://github.com/Azure/azure-rest-api-specs/pull/6683#discussion_r309897834) [](ancestors = 309897834)